hyrax-batch_ingest.gemspec

```diff
@@ -0,0 +1,23 @@
+$:.push File.expand_path("../lib", __FILE__)
+
+# Maintain your gem's version:
+require "hyrax/batch_ingest/version"
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = "hyrax-batch_ingest"
+  s.version     = Hyrax::BatchIngest::VERSION
+  s.authors     = ["Andrew Myers"]
+  s.email       = ["andrew_myers@wgbh.org"]
+  s.homepage    = "https://github.com/samvera-labs/hyrax-batch_ingest"
+  s.summary     = "Batch ingest support for Hyrax applications"
+  s.description = "Batch ingest support for Hyrax applications"
+  s.license     = "MIT"
```

Comment:
Can you change this as well to Apache v2?

Reply:
yep, will do.